packages/pwa-kit-react-sdk/src/ssr/universal/components/_app-config/index.jsx

```diff
@@ -66,6 +66,20 @@ class AppConfig extends React.Component {
         return {}
     }
 
+    /**
```

Comment:
I'm just going to drop some thoughts here about a conversation I had with @kevinxh. 

The thought brought up was, "Is the pattern of dangling static functions off of these magic components something we are moving away from, and does this just harden our current position?"

The cons are that you will at a minimum have to have an AppConfig component in your pwa projects just to config this setting, and this also makes the landscape of configuration more confusing as we have config files that the app already knows about. 

A suggest solution is to add a `basePath` config in the config file. This value can be a string or a function. This would mean that the config file has to be JS in order to use the later definition. Furthermore we can't serialize this function and we would have to add a pre-processing stage to the config to evaluate the functions before serializing. The outward API for this isn't complex but the solution is a little more complex than what we have here.


We decided that since we are doing a preview release of this work, we can see how the customer uses the current solution and then potentially invest more time making the preview GA.